docs/REST API/rest-api.md

```diff
@@ -3630,6 +3642,80 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Lock host
+
+_Available in Fleet Premium_
+
+Sends a command to lock the specified macOS, Linux, or Windows host. The host is locked once it comes online.
+
+`POST /api/v1/fleet/hosts/:id/lock`
+
+#### Parameters
+
+| Name       | Type              | In   | Description                                                                   |
+| ---------- | ----------------- | ---- | ----------------------------------------------------------------------------- |
+| id | integer | path | **Required**. ID of the host to be locked. |
+
+#### Example
+
+`POST /api/v1/fleet/hosts/123/lock`
+
+##### Default response
+
+`Status: 200`
```

Comment:
@noahtalerman @rachaelshaw looks like there is no response payload for this endpoint, typically we use `204` for that case (204 No Content).

Reply:
👍 I'll update unlock to have separate examples for macOS and Windows/Linux

```suggestion
`Status: 204`
```